test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -49,6 +57,76 @@ public sealed partial class AnimatedCarousalPage : Page
         public AnimatedCarousalPage()
         {
             this.InitializeComponent();
+
+            ElementCompositionPreview.GetElementVisual(sv).Clip = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(sv).Compositor.CreateInsetClip();
```

Comment:
Please add a comment here on why this is needed. Essentially a workaround for the comp bug that Keith mentioned.

Reply:
Done.